source/future.tex

```diff
@@ -98,20 +98,64 @@
 prefix \tcode{++} and \tcode{\dcr} expressions\iref{expr.pre.incr}
 of \tcode{volatile}-qualified arithmetic and pointer types are deprecated.
 
+\begin{example}
+\begin{codeblock}
+volatile int velociraptor;
+++velociraptor;                 // deprecated: is this a single memory operation, two, or a six-foot turkey?
+\end{codeblock}
+\end{example}
+
+
 \pnum
 Certain assignments
 where the left operand is a \tcode{volatile}-qualified non-class type
 are deprecated; see~\ref{expr.ass}.
 
+\begin{example}
+\begin{codeblock}
+int neck, tail;
+volatile int brachiosaur;
+brachiosaur = neck;             // OK, a volatile store
+tail = brachiosaur;             // OK, a volatile load
+tail = brachiosaur = neck;      // deprecated: does this access brachiosaur once or twice?
+brachiosaur += neck;            // deprecated: does this access brachiosaur once or twice?
+brachiosaur = brachiosaur + neck; // OK, a volatile load, an addition, a volatile store
+\end{codeblock}
+\end{example}
+
+
 \pnum
 A function type\iref{dcl.fct}
 with a parameter with \tcode{volatile}-qualified type or
 with a \tcode{volatile}-qualified return type is deprecated.
 
+\begin{example}
+\begin{codeblock}
+volatile amber jurassic(); // deprecated: a volatile return type has no meaning.
```

Comment:
Comments above this one don't end in a full stop. I suggest looking around where this is added and following their ways.